lib/ansible/modules/packaging/os/apt.py

```diff
@@ -153,13 +165,16 @@
     name: foo
     update_cache: yes
 
-- name: Install apache service but avoid starting it immediately
-  apt: name=apache2 state=present
-  environment:
-    RUNLEVLEL: 1
+- name: Install Apache service but avoid starting it immediately
```

Comment:
this trick still works, but on older versions of debian and derivatives, we should reflect that in description but not remove it

Reply:
If the old solution would be replaced by something more universal (like the special script) then it's not worth keeping this information in the documentation.

Reply:
the script does not work in some versions, there is a lot of overlap but i would keep both and let user figure out which one works for them

Reply:
OK, I will keep both in the examples.